src/hub.js

```diff
@@ -581,6 +614,12 @@ const onReady = async () => {
       console.error(res);
     });
 
+  channel.on("naf", data => {
+    if (NAF.connection.adapter) {
+      NAF.connection.adapter.onData(data.payload);
+    }
+  });
```

Comment:
It seems like we are still subscribing to this channel when not yet in the scene. Why not wait to subscribe? Our loading process is already extremely slow and janky and the last thing we need is a hundred websocket frames per second flying in while we do it.

Reply:
(If the answer is "we need to be subscribed for the channel for those other things", I would suggest splitting them into two channels.)

Reply:
we could potentially break things up into multiple phoenix channels, but the current hub channel is used for a variety of things before this change added naf packets to its responsibilities:

- it manages the session tracking + telemetry (a session is considered 'active' while the channel is subscribed to)
- it sends over the initial hub room information in the join reply

soon this will make a lot more sense, as the new landing page design is going to incorporate:

- presence join/leave events to the channel as users land on or leave the page
- ephemeral chat messages pre-entry
- and, likely, a full view into the game state/voices on load

Reply:
i'm of the mindset it would probably be a reasonable idea to break the naf data out into its own channel, but i don't feel like there's a forcing function yet -- this will add a sizable amount of complexity if we ever want to do anything interesting with the data flowing over that channel (like persistence)

Reply:
oh, do you mean do not subscribe to the `naf` message type? that doesn't save much other than this function dispatch I think (we will still receive the messages, they just won't be routed because there's no match on the message type.) i think i'll leave it because I think this will all be *much* more sane soon given that you will be joining the room in full on load (except will be in spectator mode.)